### PR TITLE
Streamlit UI for non technical users.

### PR DESCRIPTION
**Pull request by aashu-kattel**

### CODE REVIEW

**Comment by r-a-j-a-n** [comment]
👏👏

File: app.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                run_center_randomizer(schools_path, centers_path, prefs_path, output_file)
+
+                st.success("Center Randomization completed!")
+                st.markdown(f"Download the output file [here](/{output_file})")
+        finally:
+            # Delete temporary directory
+            shutil.rmtree(temp_dir, ignore_errors=True)
+
+if __name__ == "__main__":
+    main()
```
Comment (beingjungshahi) [inline app.py:63]: Add EOL.

**Review by chinge55 — CHANGES_REQUESTED** [review]
`streamlit` added without updating requirements.

**Comment by sumanashrestha** [comment]
thank you for introducing streamlit 
merged #44 which was inspired from this PR

**Pull request closed without merging**